app/main.py

```diff
@@ -1,3 +1,60 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> object:
```

Comment:
`__init__` returns `None`, indicate it in the return type annotation instead of `object`. Besides that, `km` could be `float` as well, add it to the type annotation for `km` using `|`.